### DISCUSSION TITLE
Suggest adding mouse button X1 and mouse button X2

### DISCUSSION THREAD

**Discussion by Back-T0** [sumatrapdfreader/sumatrapdf#3647] [Ideas]
I found that the mouse keys VK_XBUTTON1 and VK_XBUTTON2 are Navigate Back and Navigate Forward by default, is it possible to add these two keyboard shortcuts to the custom keyboard shortcuts? Because using the X1 and X2 mouse shortcuts will greatly facilitate users with multiple mouse shortcuts to flip through documents with just the mouse.

**Comment by GitHubRulesOK** [comment]
Your best bet may be to use X-mouse to reassign external Xtra buttons to other commands see https://www.highrez.co.uk/downloads/XMouseButtonControl.htm

**Comment by kjk** [comment]
I've added the XButton1 and XButton2 definitions in https://github.com/sumatrapdfreader/sumatrapdf/commit/9b499e3f513e36ab4a904af7cb5cb2418560f495

Not tested because I don't know how to trigger that.
